File: joey/week2-220719/1929. Concatenation of Array.py
class Solution1:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        for i in range(0, len(nums)):
            nums.append(nums[i])
        return nums
    
class Solution2:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        nums.extend(nums)
        return nums

File: billy/week2-220719/1929. Concatenation of Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    let list = [];

    for (i = 0; i < nums.length * 2; i += 1) {
        list.push(nums[i >= nums.length ? i - nums.length : i]);
    }

    return list;
};